webui/react/src/e2e/tests/userManagement.spec.ts

```diff
@@ -21,17 +21,15 @@ test.describe('User Management', () => {
     const userManagementPage = new UserManagement(authedPage);
     await userManagementPage.goto();
     // wait for table to be stable and select page 1
-    await userManagementPage.table.table.rows.pwLocator.count();
     const page1 = userManagementPage.table.table.pagination.pageButtonLocator(1);
-    if (await userManagementPage.table.table.pagination.pwLocator.isVisible()) {
+    await expect(async () =>
+      expect(await userManagementPage.table.table.rows.pwLocator.count()).toBeGreaterThanOrEqual(1),
```

Comment:
doesn't the `expect` need an await too? Normally the playwright expect does. Maybe just something to double check.

Reply:
no, not this one. you're thinking of
```
expect(await userManagementPage.table.table.rows.pwLocator).toHaveCount()
```
which isn't something i can use. maybe i can switch this to `waitFor` with strict mode off